examples/torch/object_detection/README.md

```diff
@@ -23,7 +23,8 @@ pip install -r examples/torch/requirements.txt
 This scenario demonstrates quantization with fine-tuning of SSD300 on VOC dataset.
 
 #### Dataset preparation
-- Download and extract VOC2007 and VOC2012 train/val and test data + devkit from [here](https://pjreddie.com/projects/pascal-voc-dataset-mirror/)
+- Download and extract in one `folder` train/val+test VOC2007 and train/val VOC2012 data from [here](https://pjreddie.com/projects/pascal-voc-dataset-mirror/)
```

Comment:
Do not use backticks to highlight anything other than the entities on a file system. Use regular "folder" instead of `folder`. Highlight `<path_to_dataset>` with backticks instead.